### ISSUE TITLE
Number widget issues

### ISSUE THREAD

**Issue by avoinea** [plone/volto#2059]
**Describe the bug**
Number widget complains about missing `_onClick, _onBlur`

**To Reproduce**
1. Go to DX Cpanel > Schema and add a new Integer / Float Filed
2. After it's added, click inside it.
3. Open console and see errors
4. Also the website becomes vary slow when you click & type something inside

**Expected behavior**
No errors and no slowness expected.

**Screenshots**
![2020-12-12-142801_1837x585_scrot](https://user-images.githubusercontent.com/323385/101983867-42dd0f80-3c86-11eb-9769-125c81dc9b57.png)

**Software (please complete the following information):**
 - OS: ArchLinux
 - Browser Chrome
 - Volto Version 10
 - Plone Version 5.2
 - Plone REST API Version 7.0.0a6.dev0

**Additional context**
```
vendors~client.chunk.js:51151 Uncaught TypeError: _onBlur is not a function
    at onBlur (vendors~client.chunk.js:51151)
    at HTMLUnknownElement.callCallback (vendors~client.chunk.js:201742)
    at Object.invokeGuardedCallbackDev (vendors~client.chunk.js:201791)
    at invokeGuardedCallback (vendors~client.chunk.js:201846)
    at invokeGuardedCallbackAndCatchFirstError (vendors~client.chunk.js:201860)
    at executeDispatch (vendors~client.chunk.js:201943)
    at executeDispatchesInOrder (vendors~client.chunk.js:201968)
    at executeDispatchesAndRelease (vendors~client.chunk.js:204832)
    at executeDispatchesAndReleaseTopLevel (vendors~client.chunk.js:204841)
    at forEachAccumulated (vendors~client.chunk.js:204813)
    at runEventsInBatch (vendors~client.chunk.js:204858)
    at runExtractedPluginEventsInBatch (vendors~client.chunk.js:205068)
    at handleTopLevel (vendors~client.chunk.js:205112)
    at batchedEventUpdates$1 (vendors~client.chunk.js:223425)
    at batchedEventUpdates (vendors~client.chunk.js:202349)
    at dispatchEventForLegacyPluginEventSystem (vendors~client.chunk.js:205122)
    at attemptToDispatchEvent (vendors~client.chunk.js:205821)
    at dispatchEvent (vendors~client.chunk.js:205743)
    at unstable_runWithPriority (vendors~client.chunk.js:251179)
    at runWithPriority$1 (vendors~client.chunk.js:212593)
    at discreteUpdates$1 (vendors~client.chunk.js:223441)
    at discreteUpdates (vendors~client.chunk.js:202360)
    at dispatchDiscreteEvent (vendors~client.chunk.js:205722)
onBlur @ vendors~client.chunk.js:51151
callCallback @ vendors~client.chunk.js:201742
invokeGuardedCallbackDev @ vendors~client.chunk.js:201791
invokeGuardedCallback @ vendors~client.chunk.js:201846
invokeGuardedCallbackAndCatchFirstError @ vendors~client.chunk.js:201860
executeDispatch @ vendors~client.chunk.js:201943
executeDispatchesInOrder @ vendors~client.chunk.js:201968
executeDispatchesAndRelease @ vendors~client.chunk.js:204832
executeDispatchesAndReleaseTopLevel @ vendors~client.chunk.js:204841
forEachAccumulated @ vendors~client.chunk.js:204813
runEventsInBatch @ vendors~client.chunk.js:204858
runExtractedPluginEventsInBatch @ vendors~client.chunk.js:205068
handleTopLevel @ vendors~client.chunk.js:205112
batchedEventUpdates$1 @ vendors~client.chunk.js:223425
batchedEventUpdates @ vendors~client.chunk.js:202349
dispatchEventForLegacyPluginEventSystem @ vendors~client.chunk.js:205122
attemptToDispatchEvent @ vendors~client.chunk.js:205821
dispatchEvent @ vendors~client.chunk.js:205743
unstable_runWithPriority @ vendors~client.chunk.js:251179
runWithPriority$1 @ vendors~client.chunk.js:212593
discreteUpdates$1 @ vendors~client.chunk.js:223441
discreteUpdates @ vendors~client.chunk.js:202360
dispatchDiscreteEvent @ vendors~client.chunk.js:205722
vendors~client.chunk.js:51152 Uncaught TypeError: _onClick is not a function
    at onClick (vendors~client.chunk.js:51152)
    at HTMLUnknownElement.callCallback (vendors~client.chunk.js:201742)
    at Object.invokeGuardedCallbackDev (vendors~client.chunk.js:201791)
    at invokeGuardedCallback (vendors~client.chunk.js:201846)
    at invokeGuardedCallbackAndCatchFirstError (vendors~client.chunk.js:201860)
    at executeDispatch (vendors~client.chunk.js:201943)
    at executeDispatchesInOrder (vendors~client.chunk.js:201968)
    at executeDispatchesAndRelease (vendors~client.chunk.js:204832)
    at executeDispatchesAndReleaseTopLevel (vendors~client.chunk.js:204841)
    at forEachAccumulated (vendors~client.chunk.js:204813)
    at runEventsInBatch (vendors~client.chunk.js:204858)
    at runExtractedPluginEventsInBatch (vendors~client.chunk.js:205068)
    at handleTopLevel (vendors~client.chunk.js:205112)
    at batchedEventUpdates$1 (vendors~client.chunk.js:223425)
    at batchedEventUpdates (vendors~client.chunk.js:202349)
    at dispatchEventForLegacyPluginEventSystem (vendors~client.chunk.js:205122)
    at attemptToDispatchEvent (vendors~client.chunk.js:205821)
    at dispatchEvent (vendors~client.chunk.js:205743)
    at unstable_runWithPriority (vendors~client.chunk.js:251179)
    at runWithPriority$1 (vendors~client.chunk.js:212593)
    at discreteUpdates$1 (vendors~client.chunk.js:223441)
    at discreteUpdates (vendors~client.chunk.js:202360)
    at dispatchDiscreteEvent (vendors~client.chunk.js:205722)
```

**Comment by ichim-david** [comment]
This issue is now elevated from a simple console warning to a full page crash
<img width="885" alt="error-number-widget" src="https://user-images.githubusercontent.com/152852/101986159-01a02c00-3c95-11eb-8aad-de5f461433f3.PNG">

**Comment by ichim-david** [comment]
@avoinea added https://github.com/plone/volto/pull/2062 for this issue, test and check the fix

**Comment by ichim-david** [comment]
Fix is now in core and released within 10.4.3, we can therefore close the issue.
